### DISCUSSION TITLE
How to use session persistance

### DISCUSSION THREAD

**Discussion by Pedro-0550** [gladly-team/next-firebase-auth#363] [Q&A] [answered]
I am trying to use session persistance so the user can choose if he wants to keep logged in or not, but next-firebase-auth thinks the user is still logged in after the session is over, is there a way to fix this?

I am using the canary version.

**Comment by kmjennison** [comment, marked as answer]
Yes. Be sure your cookies settings and Firebase persistence settings are aligned. See: https://github.com/gladly-team/next-firebase-auth/discussions/54#discussioncomment-309191
